core/src/main/java/com/digitalpebble/stormcrawler/parse/filter/SignatureCopyParseFilter.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to DigitalPebble Ltd under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * DigitalPebble licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.digitalpebble.stormcrawler.parse.filter;
+
+import java.util.Map;
+
+import org.w3c.dom.DocumentFragment;
+
+import com.digitalpebble.stormcrawler.Metadata;
+import com.digitalpebble.stormcrawler.parse.ParseData;
+import com.digitalpebble.stormcrawler.parse.ParseFilter;
+import com.digitalpebble.stormcrawler.parse.ParseResult;
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * Adds a copy of the current signature to metadata.
+ */
+public class SignatureCopyParseFilter extends ParseFilter {
```

Comment:
No need for a separate ParseFilter, let's add a config element to the existing Signature one so that it makes a copy of any preexisting value before generating a new one.